src/patternfly/components/Tabs/tabs.scss

```diff
@@ -250,6 +253,7 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
 
   // Vertical
   &.pf-m-vertical {
+    --pf-c-tabs--Width: var(--pf-c-tabs--m-vertical--Width);
```

Comment:
Do you think this could be breaking? Here's a probable use case where adding the width could break the layout. Though I agree this should have a block layout and consume the width of whatever it's in, I wonder if this could break people.

Reply:
Yeah, I suppose it could be. All we need to change here is `flex-grow: 1`. I'll revert.